README.md

```diff
@@ -94,6 +94,10 @@ Rewrite the prefix to the specified string. Default: `''`.
 
 A `preHandler` to be applied on all routes. Useful for performing actions before the proxy is executed (e.g. check for authentication).
 
+### skipContentParsing
+
+The default value of this option is true. The ContentTypeParsers are removed by default for proxy but incase you want to access the body to take certain decision like authentication or authorization then you can set this option to false.
```

Comment:
```suggestion
Setting this option to `false` will disable pass through of proxied payloads. When setting this to `false`, it is left up to the implementor to handle proxied payloads correctly.
```

Reply:
There should also be an example here showing what handling of a payload looks like.

Reply:
Hi @jsumners,
I don't think that making this option false will disable pass through of proxied payloads. It would even then pass the payload to the upstream.


If you are thinking of some ContentType which would create problems in parsing then please do give example because I might not know about it..